src/styles/index.scss

```diff
@@ -1,3 +1,108 @@
 body {
   margin: 0;
 }
+
+.stopwatch {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.watch {
+  position: relative;
+  height: 80vmin;
+  width: 80vmin;
+  content: '';
+  border: 1vmin dotted #000;
+  border-radius: 100%;
+}
+
+.arrow-min {
+  position: absolute;
+  bottom: 50%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  content: '';
+  height: 20vmin;
+  width: 3vmin;
+  background-color: #0700ff;
+  transform-origin: bottom;
+  animation: min 3600s linear infinite;
+}
```

Comment:
You have doublicated properties which you reuse in different selectors. To remove doublication you can use [scss placeholders](https://sass-lang.com/documentation/style-rules/placeholder-selectors/)

Reply:
Please fix it for all of the selectors below